Source/Scene/Cesium3DTileset.js

```diff
@@ -1761,6 +1765,51 @@ Cesium3DTileset.prototype.loadTileset = function (
   return rootTile;
 };
 
+/**
+ * Make a {@link Cesium3DTile} for a specific tile. If the tile has the
+ * 3DTILES_implicit_tiling extension, it creates a placeholder tile instead
+ * for lazy evaluation of the implicit tileset.
+ *
+ * @param {Cesium3DTileset} tileset The tileset
+ * @param {Resource} baseResource The base resource for the tileset
+ * @param {Object} tileHeader The JSON header for the tile
+ * @param {Cesium3DTile} [parentTile] The parent tile of the new tile
+ * @returns {Cesium3DTile} The newly created tile
+ *
+ * @private
+ */
+function makeTile(tileset, baseResource, tileHeader, parentTile) {
+  if (has3DTilesExtension(tileHeader, "3DTILES_implicit_tiling")) {
+    var implicitTileset = new ImplicitTileset(baseResource, tileHeader);
+    var rootCoordinates = new ImplicitTileCoordinates({
+      subdivisionScheme: implicitTileset.subdivisionScheme,
+      level: 0,
+      x: 0,
+      y: 0,
+      // The constructor will only use this for octrees.
+      z: 0,
```

Comment:
Since everything else that creates `ImplicitTileCoordinates` leaves z undefined if octree, maybe this should do the same. 

This would be less work for `derivedBoundingVolume`. And even though it yields the same result whether defined or not, it may introduce some floating point error due to the math.

Reply:
Never mind, `implicitTileCoordinates` will keep it undefined if the tiling scheme is not OCTREE